src/components/common/NewActionButton/NewActionButton.tsx

```diff
@@ -281,12 +290,14 @@ const NewActionButton = (/* { colony, ethDomainId }: Props */) => {
           <Button
             appearance={{ theme: 'primary', size: 'large' }}
             text={MSG.newAction}
-            onClick={() => startWizardFlow('dashboard.ColonyActionsDialog')}
+            onClick={() =>
```

Comment:
This view needs to be reused by `NewDecisionButton` for Simple Decisions, but naturally doesn't need most of the accompanying logic. 

I created [`DialogButton`](https://github.com/JoinColony/colonyCDapp/blob/port/188-remove-draft-decision-modal/src/components/shared/Button/DialogButton.tsx) to extract the view for reuse between both. 

Fancy copying that over and using it here?